src/components/RegionSelector/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { Grid, useTheme } from '@mui/material';
```

Comment:
@constanzauanini i created a shared component for region selection - used in org create and edit.
not using the DropdownAutocomplete yet.. something is not working there.
will follow up separately